help/en/releasenotes/current-draft-note.shtml

```diff
@@ -116,7 +116,7 @@
 
         <h4><a href="http://openlcb.org">OpenLCB</a> / LCC</h4>
             <ul>
-                <li></li>
+                <li>Add a factory reset option to the configuration More... menu.</li>
```

Comment:
Is that feature included in the openlcb.jar file? Since that file is the only important change in this PR.

Reply:
Yes.  It's PR 227 there.